test/traffic_cop/traffic_cop_test.cpp

```diff
@@ -144,6 +145,28 @@ TEST_F(TrafficCopTests, BasicTest) {
   }
 }
 
+// NOLINTNEXTLINE
+TEST_F(TrafficCopTests, OrderTest) {
+  try {
+    pqxx::connection connection(fmt::format("host=127.0.0.1 port={0} user={1} sslmode=disable application_name=psql",
+                                            port_, catalog::DEFAULT_DATABASE));
+
+    pqxx::work txn1(connection);
+    txn1.exec("DROP TABLE IF EXISTS TableB");
+    txn1.exec("CREATE TABLE TableB (id INT PRIMARY KEY, data TEXT, num INT);");
+    txn1.exec("INSERT INTO TableB VALUES (1, 'abc', 3);");
+    txn1.exec("INSERT INTO TableB VALUES (2, 'abc', 2);");
+    txn1.exec("INSERT INTO TableB VALUES (3, 'abc', 1);");
+    pqxx::result r = txn1.exec("SELECT id,data,num FROM TableB ORDER BY num");
```

Comment:
How does this actually test ORDER BY column#? It is ordering by the column name.

You should execute two queries and check that output is the same:

1. `SELECT id,data,num FROM TableB ORDER BY num`
2. `SELECT id,data,num FROM TableB ORDER BY 3`

Reply:
@apavlo , Dear Professor Apavlo. I've modified the test case. And when I execute the test separately, it could succeed. But when I build the "unittest" target, the test case fails.
![gtest1](https://user-images.githubusercontent.com/22066046/86308630-636f5b00-bc4c-11ea-98dc-b60885fb0353.png)
![gtest2](https://user-images.githubusercontent.com/22066046/86308640-6a966900-bc4c-11ea-8ce7-6afa27351d94.png)

Reply:
Below is the unit test case.
```
TEST_F(TrafficCopTests, OrderTest) {
  try {
    pqxx::connection connection(fmt::format("host=127.0.0.1 port={0} user={1} sslmode=disable application_name=psql",
                                            port_, catalog::DEFAULT_DATABASE));

    pqxx::work txn1(connection);
    txn1.exec("DROP TABLE IF EXISTS TableA");
    txn1.exec("CREATE TABLE TableA (id INT PRIMARY KEY, data TEXT, num INT);");
    txn1.exec("INSERT INTO TableA VALUES (1, 'abc', 3);");
    txn1.exec("INSERT INTO TableA VALUES (2, 'abc', 2);");
    txn1.exec("INSERT INTO TableA VALUES (3, 'abc', 1);");
    pqxx::result r1 = txn1.exec("SELECT id,data,num FROM TableA ORDER BY num");
    pqxx::result r2 = txn1.exec("SELECT id,data,num FROM TableA ORDER BY 3");
    int value1 = 0;
    r1.begin().at(0).to(value1);
    int value2 = -1;
    r2.begin().at(0).to(value2);
    EXPECT_EQ(value1, value2);
    txn1.commit();
  } catch (const std::exception &e) {
    EXPECT_TRUE(false);
  }
}
```